third_party/xla/xla/service/gpu/gpu_layout_assignment.cc

```diff
@@ -55,7 +55,7 @@ limitations under the License.
 #include "tsl/platform/errors.h"
 #include "tsl/platform/statusor.h"
 
-#include "tsl/util/env_var.h"
+#include "xla/tsl/util/env_var.h"
```

Comment:
same. this one is actually an old one we haven't upstream for a long while. https://github.com/ROCm/tensorflow-upstream/commit/dfb7dd0b2f7e6c6bbcd3f1dbc9e299be9554b4a7 could you upstream this as well?

Reply:
Same as above, PR [#11307](https://github.com/openxla/xla/pull/11307)